src/incentives/AffinePassBridge.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {CCIPReceiver} from "@chainlink/contracts-ccip/src/v0.8/ccip/applications/CCIPReceiver.sol";
+import {Client} from "@chainlink/contracts-ccip/src/v0.8/ccip/libraries/Client.sol";
+import {IRouterClient} from "@chainlink/contracts-ccip/src/v0.8/ccip/interfaces/IRouterClient.sol";
+import {AffinePass} from "./affine-pass.sol";
```

Comment:
This import does not work. The AffinePass PR needs to be merged into master, then we can merge master into this PR.